src/components/OwnerMessage/OwnerMessage.jsx

```diff
@@ -127,7 +127,8 @@ function OwnerMessage({
     <div className="message-container">
       {ownerMessage ? getContent(ownerMessage) : getContent(ownerStandardMessage)}
 
-      {user.role === 'Owner' && (
+      {(user.role === 'Owner' ||
+        user.permissions.frontPermissions.includes('editHeaderMessage')) && (
```

Comment:
```suggestion
      {(props.hasPermission('editHeaderMessage')) && (
```
Should use `hasPermission()` to also include role permissions in case that's ever needed.